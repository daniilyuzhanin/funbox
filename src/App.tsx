import { createTheme, ThemeProvider, Theme } from '@mui/material/styles';
import { Layout } from 'common/components/Layout';
import { ProductCard } from 'common/components/ProductCard';
import React from 'react';
import { BrowserRouter } from 'react-router-dom';

declare module '@mui/styles/defaultTheme' {
  interface DefaultTheme extends Theme {}
}

const theme = createTheme({
  palette: {
    primary: {
      light: '#fff',
      contrastText: '#000000',
      main: '#1698d9',
      dark: '#002884',
    },
    secondary: {
      light: '#F2F2F2',
      contrastText: '#666666',
      main: '#11cb5f',
    },
  },
  spacing: 2,
  typography: {
    fontFamily: '"Trebuchet MS", "Arial", sans-serif, serif',
    h1: {
      fontSize: 48,
      fontWeight: 700,
    },
    h2: {
      fontSize: 36,
      fontFamily: "'Exo 2', sans-serif",
    },
    h3: {
      fontSize: 24,
      fontWeight: 700,
    },
    subtitle1: {
      fontSize: 22,
    },
  },
});

const App = () => (
  <BrowserRouter>
    <ThemeProvider theme={theme}>
      <Layout>
        <ProductCard filler="с фуа-гра" weight="0,5" />
        <ProductCard filler="с рыбой" weight={2} />
        <ProductCard filler="с курой" weight={5} />
      </Layout>
    </ThemeProvider>
  </BrowserRouter>
);

export default App;
