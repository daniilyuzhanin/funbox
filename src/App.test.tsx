import { render } from '@testing-library/react';
import React from 'react';

import App from './App';

test('renders home page', () => {
  const { container } = render(
    <App />,
  );
  expect(container).toMatchSnapshot();
});
