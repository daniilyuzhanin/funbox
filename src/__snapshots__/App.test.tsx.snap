// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders home page 1`] = `
<div>
  <div
    class="MuiGrid-root MuiGrid-container makeStyles-wrapper-1 css-11lq3yg-MuiGrid-root"
  >
    <div
      class="MuiGrid-root MuiGrid-container MuiGrid-direction-xs-column makeStyles-layout-2 css-1vpwcmr-MuiGrid-root"
    >
      <div
        class="MuiGrid-root MuiGrid-item makeStyles-layoutTitle-3 css-10omzie-MuiGrid-root"
      >
        Ты сегодня покормил кота?
      </div>
      <div
        class="MuiGrid-root MuiGrid-container css-1lym95h-MuiGrid-root"
      >
        <div
          class="MuiGrid-root MuiGrid-item css-13i4rnv-MuiGrid-root"
        />
      </div>
    </div>
  </div>
</div>
`;
